### ISSUE TITLE
Mobile A11y Testing 11/1

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-menu-behavior#35]
On Android with Talkback, when an item is selected, I have no spoken feedback telling me about the state, e.g. selected or unselected.  This context is missing for the user.
